README.rst

```diff
@@ -1,662 +1,26 @@
-    network_graph = NetworkGraphView.as_view()
-    network_graph_history = NetworkGraphHistoryView.as_view()
-    receive_topology = ReceiveTopologyView.as_view()
-
-API URLs
-^^^^^^^^
-
-If you are not making drastic changes to the api views, you can avoid duplicating the URL logic by using the ``get_api_urls`` function. Put this in your api ``urls.py``:
-
-.. code-block:: python
-
-    # your app.api.urls
-    from django_netjsongraph.utils import get_api_urls
-    from . import views
-
-    urlpatterns = get_api_urls(views)
-
-Extending Visualizer Views
-^^^^^^^^^^^^^^^^^^^^^^^^^^
-If your use case doesn't vary much from the base, you may also want to try to reuse the Visualizer views:
-
-.. code-block:: python
-
-    # your app.visualizer.views
-    from ..models import Topology
-    from .generics import BaseTopologyDetailView, BaseTopologyListView
-
-
-    class TopologyListView(BaseTopologyListView):
-        topology_model = Topology
-
-
-    class TopologyDetailView(BaseTopologyDetailView):
-        topology_model = Topology
-
-
-    topology_list = TopologyListView.as_view()
-    topology_detail = TopologyDetailView.as_view()
-
-
-Visualizer URLs
-^^^^^^^^^^^^^^^
-If you are not making any drastic changes to visualizer views, you can avoid duplicating the URL logic by using ``get_visualizer_urls`` function. Put this in your visualizer ``urls.py``
-
-.. code-block:: python
-
-    # your app.visualizer.urls
-    from django_netjsongraph.utils import get_visualizer_urls
-    from . import views
-
-    urlpatterns = get_visualizer_urls(views)
-
-Extending AppConfig
-^^^^^^^^^^^^^^^^^^^
-
-You may want to reuse the ``AppConfig`` class of *django-netjsongraph* too:
-
-.. code-block:: python
-
-    from django_netjsongraph.apps import DjangoNetjsongraphConfig
-
-    class MyOwnConfig(DjangoNetjsongraphConfig):
-        name = 'yourapp'
-        label = 'yourapp'
-
-Installing for development
---------------------------
-
-Install sqlite:
-
-.. code-block:: shell
-
-    sudo apt-get install sqlite3 libsqlite3-dev
-
-Install your forked repo:
-
-.. code-block:: shell
-
-    git clone git://github.com/<your_fork>/django-netjsongraph
-    cd django-netjsongraph/
-    python setup.py develop
-
-Install test requirements:
-
-.. code-block:: shell
-
-    pip install -r requirements-test.txt
-
-Create database:
+1. Take a backup of your database.
+2. Create JSON backup of data that's required for migration with the following command:
 
 .. code-block:: shell
 
-    cd tests/
-    ./manage.py migrate
-    ./manage.py createsuperuser
+    $ cd django-netjsongraph # Go to the django-netjsongraph repository (or installed app)
+    $ python tests/manage.py dumpdata auth.user > user.json
+    $ python tests/manage.py dumpdata auth.group > group.json
+    $ python tests/manage.py dumpdata django_netjsongraph > netjsongraph.json
+    $ python tests/manage.py dumpdata auth.permission > permission.json
+    $ pwd # copy output of the command
```

Comment:
I'd remove the leading dollar sign, so that it can easily be copied and pasted into a terminal